client/src/components/common/InfoCard.js

```diff
@@ -2,17 +2,22 @@ import React from 'react';
 import {
   Card, CardBody, CardText, CardTitle,
 } from 'reactstrap';
+import PropTypes from 'prop-types';
 
-const InfoCard = () => {
+const InfoCard = (props) => {
```

Comment:
You can destructure the props here only if you are not passing the entire/ spreading props in any other component.